autotst/conformer/systematic.py

```diff
@@ -197,7 +197,18 @@ def opt_conf(conformer, calculator, i):
             label = conformer.smiles
 
         if isinstance(calc, FileIOCalculator):
-            calculator.directory = "systematic_{}".format(str(i))
+            calculator.directory = os.path.join(
+                'conformer_logs',
```

Comment:
This will store these log file where ever you ran them, right? Would it make more sense to store them where the Gaussian log files will be stored so they're all in one place? `directory/species/smiles/conformer_logs/systematic/smiles_i`?